### PR TITLE
closure proto library for example protos

### PR DESCRIPTION
**Pull request by jameswex**
Build TF example protos using closure_proto_library so they can be used by TesnsorBoard TS/JS code in the future.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,13 +1,18 @@
 workspace(name = "org_tensorflow")
 
-http_archive(
+## DO NOT SUBMIT
```
Comment (jart) [inline WORKSPACE]:
```py
http_archive(
    name = "io_bazel_rules_closure",
    sha256 = "a38539c5b5c358548e75b44141b4ab637bba7c4dc02b46b1f62a96d6433f56ae",
    strip_prefix = "rules_closure-dbb96841cc0a5fb2664c37822803b06dab20c7d1",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_closure/archive/dbb96841cc0a5fb2664c37822803b06dab20c7d1.tar.gz",
        "https://github.com/bazelbuild/rules_closure/archive/dbb96841cc0a5fb2664c37822803b06dab20c7d1.tar.gz",  # 2018-04-13
    ],
)
```

File: tensorflow/core/BUILD [hunk 2]
```diff
@@ -149,6 +149,7 @@ load(
     "//third_party/mkl:build_defs.bzl",
     "if_mkl",
 )
+load("@io_bazel_rules_closure//closure:defs.bzl","closure_proto_library")
```
Comment (jart) [inline tensorflow/core/BUILD]: [nit] space after comma so we don't get unrelated buildifier edits later.

**Review by jart — CHANGES_REQUESTED** [review]

File: WORKSPACE (hunk 1 above)
Comment (jameswex) [reply WORKSPACE]: done

File: tensorflow/core/BUILD (hunk 2 above)
Comment (jameswex) [reply tensorflow/core/BUILD]: done

**Review by jameswex — COMMENTED** [review]
made updates as suggested by jart

**Review by jart — APPROVED** [review]

**Pull request merged**